Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -11526,7 +11526,7 @@ - (NSDictionary *)_contentsOfUserInterfaceItem:(NSString *)userInterfaceItem
 #if HAVE(LINK_PREVIEW)
     if ([userInterfaceItem isEqualToString:@"contextMenu"]) {
         auto itemTitles = adoptNS([NSMutableArray<NSString *> new]);
-        [_contextMenuInteraction updateVisibleMenuWithBlock:[&itemTitles](UIMenu *menu) -> UIMenu * {
```

Comment:
We should consider renaming this variable, given that `self.contextMenuInteraction` doesn't always return it anymore.

Reply:
Yeah, that makes sense (I also wonder if we can simply remove this variable altogether, if `HAVE(TEXT_INTERACTION_WITH_CONTEXT_MENU_INTERACTION)`). @mwyrzykowski, do you know if there are cases on iOS 16 where we still rely on `_contextMenuInteraction` rather than the text interaction assistant's context menu interaction.

Reply:
yes it should be fine for iOS 16 to remove it

Reply:
Filed https://webkit.org/b/243117 for this — since there isn't an urgent need to do this right away, let's wait until we drop support for iOS 15 in EWS/the OpenSource WebKit build.